tutorials/iot/ubuntu-kiosk/x11-kiosk.md

```diff
@@ -316,31 +322,39 @@ summary: glxgears example kiosk
 description: |
   glxgears example X11 kiosk, using Xwayland and Wayland
 base: core18
-confinement: devmode
+confinement: strict
```

Comment:
Should this change be made, since it still requires `--devmode` to work?

Reply:
Yes, it's consistent with the yaml used earlier in the tutorial.